lib/method_cache.rb

```diff
@@ -92,6 +92,15 @@ def self.remove_steps(file)
       end
     end
 
+    def self.is_file_cached(file)
```

Comment:
Test

Reply:
Test is here https://github.com/getgauge/gauge-ruby/pull/101/files#diff-d59828c1a1514452076fc2b9442f3356R245